packages/ui5-application-writer/templates/README.md

```diff
@@ -0,0 +1,40 @@
+## Application Details
+|               |
+| ------------- |
+|**Generation Date and Time**<br><%= genDate %>|
+|**App Generator**<br><%= genId %>|
+|**App Generator Version**<br><%= genVersion %>|
+|**Generation Platform**<br><%= genPlatform %>|
+|**Template Used**<br><%= templateLabel %>|
+|**Service Type**<br><%= dataSourceLabel %>|<% if(metadataFilename) { %>
+|**Metadata File**<br><%= metadataFilename %><% } else { %>
+|**Service URL**<br><%= serviceUrl %><%}%>
```

Comment:
I've noticed that in all of our generated projects, the Service URL is always missing the trailing | and as a result it is not formatted like the others. Is this something we can try and fix?

Reply:
yes will add - Thanks